### PR TITLE
[Bug Fix] Undead Should Never Flee

### PR DESCRIPTION
**Pull request by fryguy503**
Added a hard return on Undead Body Types. They should never flee under any circumstance.

### CODE REVIEW

File: zone/fearpath.cpp [hunk 1]
```diff
@@ -49,6 +49,11 @@ void Mob::CheckFlee()
 		return;
 	}
 
+	// Undead do not flee
+	if(GetBodyType() == BT_Undead) {
```
Comment (Akkadius) [inline zone/fearpath.cpp:53]: `if (` vs `if(`

**Review by Akkadius — APPROVED** [review]

**Review by joligario — COMMENTED** [review]
I feel that the immune to flee flag could handle this. What if a custom server wants them to flee?

**Comment by fryguy503** [comment]
I can rule it?

There are 5886 npc's that are BodyType Undead (3) but only 3948 of those include "21" in their special_abilities for immune to flee.

**Comment by Akkadius** [comment]
I'd rather we lean into sane defaults for this one and have undead just not flee

**Comment by joligario** [comment]
> I'd rather we lean into sane defaults for this one and have undead just not flee

It’s not a default if it is the only option

**Pull request merged**